data/generalhw_scripts/flash_sd_rootless.sh

```diff
@@ -13,7 +13,7 @@ if [ "$#" -ne 4 ]; then
     exit 1;
 fi
 
-# Get destination: /storage/
+# Get destination (tmpdir to extract image): /storage/
 destination_folder=$1
```

Comment:
`destination_folder` is not used anymore apparently, and thus, could be removed.

Reply:
Right. Updated script and docs.